### PR TITLE
Donor page

### PR DESCRIPTION
**Pull request by brendobrendo**
#### Please check if the PR fulfills these requirements

- [X] The changes don't come from your master branch. ([Source](https://blog.jasonmeridth.com/posts/do-not-issue-pull-requests-from-your-master-branch/)) 

---

#### What kind of change does this PR introduce? (bug fix, feature, docs update, ...)

All Donors Page

#### What is the new behavior? (if this is a feature change)
Renders page

#### Does this PR introduce a breaking change? (What changes might users need to make in their application due to this PR?)
No

#### Other information
TODO items noted

### CODE REVIEW

**Comment by renburnett** [comment]
@brendobrendo hey! no biggie, but I noticed that this PR template was never filled out-- I know its a bit of a pain but at the very least we need to be linking the trello card so the reviewer knows what bugfixes/features they are looking for. 

So going forward, if you are in a rush and/or dont want to fill out the whole template, feel free to just kill all the text and paste the trello card link 🔗 ✔️

**Comment by brendobrendo** [comment]
@renburnett my bad! I rushed that at the end of last week's session. Will do that going forward. Updated the card.

**Comment by renburnett** [comment]
> @renburnett my bad! I rushed that at the end of last week's session. Will do that going forward. Updated the card.

no prob! cheers ☺️

**Comment by renburnett** [comment]
> @renburnett my bad! I rushed that at the end of last week's session. Will do that going forward. Updated the card.

@brendobrendo no prob! cheers ☺️

**Comment by brendobrendo** [comment]
@renburnett Did anything besides the PR template need to be changed?

File: src/Pages/AllDonorsPage/index.js [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+  {
+    name: 'Jason Derulo 12',
+    businessName: 'Autotunes, Inc',
+    dateSubmitted: '2023/01/17',
+    status: 'active',
+  },
+];
+
+function AllDonorsPage() {
+  const entriesPerPage = 10;
```
[the remark this answers is not in the document]
Comment (tackc) [reply src/Pages/AllDonorsPage/index.js:89]: We can add this logic later if so decided

**Review by tackc — APPROVED** [review]
If we decide to give a user the ability to choose how many entries per page to view, we can add it.

Eventually, once we implement the pagy gem, we can remove any related pagination logic that is no longer needed

**Pull request merged**